packages/dart/npt_flutter/lib/features/onboarding/widgets/onboarding_button.dart

```diff
@@ -282,6 +281,8 @@ class _OnboardingButtonState extends State<OnboardingButton> {
               atClientPreference: atClientPrefernce,
             ),
           );
+          // If the user used APKAM, we do not have to back up the keys after onboarding
+          App.navState.currentContext!.read<BackupKeyCubit>().setBackupKeyStatus(true);
```

Comment:
We shouldn't tell the user to backup at all for onboarding by file either, not just apkam. If they used a file to activate the app then they don't need to backup another copy of the same file they just used.